lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -159,34 +169,18 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     Tensor *w = getTensorByName(op.input(1));
 
     // Transpose the weights to the right format. Glow expects to read the
-    // weights in the format CRSK. Caffe2 stores the operators as KCRS.
-    // C - output_depth, R - filter_height, S - filter_width, K - input_depth.
+    // weights in the format NHWC.
     Tensor wtag;
-    w->transpose(&wtag, NCHW2NHWC);
+    if (typeName != "Conv" && order == "NHWC")
+      wtag.assign(w);
+    else
+      w->transpose(&wtag, NCHW2NHWC);
 
     // The structure of the conv weigts is: NHWC. We take the C, which is the
     // number of filters. We use this value to calculate the size of the bias
     // if it is not specified.
     size_t depth = wtag.dims()[0];
 
-    // Construct the Filter field.
```

Comment:
Why did we remove this part of the code?

Reply:
Rewrite it later based on the filter's type -- could be int8 or float type